mtp_cashbook/apps/disbursements/views.py

```diff
@@ -216,7 +216,8 @@ def get_nomis_balances(self):
             pass
 
     def get(self, request, **kwargs):
-        if request.is_ajax():
+        is_ajax = request.headers.get('x-requested-with') == 'XMLHttpRequest'
```

Comment:
if they removed it, does that mean it doesn't work in modern browsers anymore? this functionality may need to change; like perhaps there needs to be a separate view for the ajax-y version…